test/README.md

```diff
@@ -0,0 +1,64 @@
+# HOW TO USE IT
+
+This document describe how to use the benchmarking scripts. 
+
+
+#### Folder tree
```

Comment:
You can delete this title and add the tree in "how to use it" section. However, you must add a description (small paragraph which explains the tree).